docs/src/poll.rst

```diff
@@ -106,11 +106,24 @@ API
     .. note::
         Though UV_DISCONNECT can be set, it is unsupported on AIX and as such will not be set
         on the `events` field in the callback.
+        
+    .. note::
+        If one of the events UV_READABLE or UV_WRITABLE are set, the callback will be called over-and-over again, as long as the given fd/socket remains readable or writable accordingly. Particullary in each of the following scenerios:
```

Comment:
hi! those constants are refrenced all over this file without backticks. should we put them all between backticks?